util/progress/progressui/init.go

```diff
@@ -0,0 +1,40 @@
+package progressui
+
+import (
+	"os"
+	"runtime"
+
+	"github.com/morikuni/aec"
+)
+
+var colorRun aec.ANSI
+var colorCancel aec.ANSI
+var colorWarning aec.ANSI
+var colorError aec.ANSI
+
+func init() {
+	// As recommended on https://no-color.org/
+	if _, ok := os.LookupEnv("NO_COLOR"); ok {
+		colorRun = nil
```

Comment:
Why set them if they are nil by default anyway.

Reply:
I was just trying to be explicit and make what was happening a bit more obvious, but I went ahead and replaced it with a comment.